index.html

```diff
@@ -358,66 +316,149 @@
       </section>
       <section>
         <h2>
-          <code>CanvasImageSource</code> typedef
+          <code>DepthData</code> interface
```

Comment:
That's a fair point - linguistically I like the symmetry of ImageData/DepthData - but since there's an existing convention I'd be happy if we ended up with ImageData/DepthMap.